JetStream.css

```diff
@@ -296,11 +296,14 @@ a.button {
 
 }
 
-.benchmark h3, .benchmark h4, .benchmark .result, .benchmark label {
+ .benchmark h4, .benchmark .result, .benchmark label {
```

Comment:
There's an extra space here probably worth cleaning up for consistency.

Reply:
👍 
oh I should probably copy over the `npm run format` commands from Speedometer in another PR.

Reply:
Still a space here.

Reply:
ups, I'm blind to formatting :P